props/saber_BC_buttons.h

```diff
@@ -913,7 +980,17 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
         SaberBase::DoEffect(EFFECT_POWERSAVE, 0);
         return true;
       } else {
+  #ifndef BC_SWAP
+        sound_library_.SayZoomingIn();
+        if (!spam_blast_) {
```

Comment:
this whole if statement can be written as:

spam_blast_ = !spam_blast_;

Reply:
ok so this just flips the bool